src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -93,7 +93,7 @@ export const terminalConfiguration: IConfigurationNode = {
 		'terminal.integrated.altClickMovesCursor': {
 			description: localize('terminal.integrated.altClickMovesCursor', "If enabled, alt/option + click will move the prompt cursor to position underneath the mouse. This may not work reliably depending on your shell."),
 			type: 'boolean',
-			default: true
+			default: false
```

Comment:
We want true by default, but overriding to false when `multiCursorModifier` is changed.